src/App.vue

```diff
@@ -1,10 +1,46 @@
 <template>
-  <RouterView class="app"></RouterView>
+  <RouterView class="app" @error="handleError"></RouterView>
 </template>
 
 <script>
+import Vue from 'vue'
+import Alert from '/components/atomic/Alert'
+import CubeError from '/error'
+
 export default {
   name: 'App',
+  errorCaptured(err) {
+    this.handleError(err)
+    return false
+  },
+  methods: {
+    handleError(err) {
+      if (
+        err.message.startsWith('Network error') ||
+        (err.graphQLERrors &&
```

Comment:
I think there's a typo here? `err.graphQLERrors` instead of `err.graphQLErrors`

Reply:
fixed with 5f8c89c